Mesh_3/include/CGAL/Mesh_polyhedron_3.h

```diff
@@ -215,6 +217,117 @@ struct Mesh_polyhedron_3
   typedef type Type;
 };
 
+
+template <typename Gt, typename Patch_id>
+struct Patch_id_pmap {
```

Comment:
@maxGimeno: There are two `Patch_id_pmap` in this PR, and I have already worked with you, in another PR (the one about `Surface_mesh` and `Mesh_3`) to unify all those property maps into a better reusable implementation.